File: mifosng-android/src/main/java/com/mifos/services/data/ChargesPayload.java
package com.mifos.services.data;

/**
 * Created by nellyk on 2/15/2016.
 */

/*
 * This project is licensed under the open source MPL V2.
 * See https://github.com/openMF/android-client/blob/master/LICENSE.md
 */

public class ChargesPayload {

    private Integer chargeId;
    private Integer clientId;
    private String amount;
    private String locale;
    private String dueDate;
    private String dateFormat;

    public String getDateFormat() {
        return dateFormat;
    }

    public void setDateFormat(String dateFormat) {
        this.dateFormat = dateFormat;
    }

    public Integer getClientId() {
        return clientId;
    }

    public void setClientId(Integer clientId) {
        this.clientId = clientId;
    }

    public String getLocale() {
        return locale;
    }

    public void setLocale(String locale) {
        this.locale = locale;
    }

    public String getDueDate() {
        return dueDate;
    }


    public void setDueDate(String dueDate) {
        this.dueDate = dueDate;
    }


    public Integer getChargeId() {
        return chargeId;
    }

    public void setChargeId(Integer chargeId) {
        this.chargeId = chargeId;
    }

    public String getAmount() {
        return amount;
    }

    public void setAmount(String amount) {
        this.amount = amount;
    }
}

File: core/database/src/main/java/com/mifos/core/data/ChargesPayload.kt
package com.mifos.core.data

/**
 * Created by nellyk on 2/15/2016.
 */
/*
 * This project is licensed under the open source MPL V2.
 * See https://github.com/openMF/android-client/blob/master/LICENSE.md
 */
class ChargesPayload {
    var chargeId: Int? = null
    var clientId: Int? = null
    var loanId: Int? = null
    var amount: String? = null
    var locale: String? = null
    var dueDate: String? = null
    var dateFormat: String? = null
}